src/bin/webpack-validator.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+'use strict'
+const fs = require('fs')
+const path = require('path')
+const program = require('commander')
+const log = require('npmlog')
+const validate = require('../')
+const schema = require('../').schema
+let configFile
+
+program
+  .arguments('<configFileName>')

```

Comment:
This should default to `webpack.config.js`, like the webpack binary does as well, no?